pallets/admin-utils/src/weights.rs

```diff
@@ -430,7 +431,12 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 		Weight::from_parts(47_279_000, 4697)
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
-	}
+	}		
+	fn sudo_set_owner_cut() -> Weight {
```

Comment:
I see that you haven't updated the benchmarking scripts.
How did you get the weight values here?